lib/redis/cluster.rb

```diff
@@ -69,7 +69,12 @@ def with_reconnect(val = true, &block)
     end
 
     def call(command, &block)
```

Comment:
Should we also care the following method using in `#call_loop`, `#call_pipeline` and so on?

https://github.com/redis/redis-rb/blob/78bf94051d8176a1a5721b8647a1ceb730119ab2/lib/redis/cluster.rb#L215-L229

Reply:
good point, done 👍